query.go

```diff
@@ -67,6 +67,7 @@ func GetQuery(query backend.DataQuery) (*Query, error) {
 	model := &Query{}
 
 	if err := json.Unmarshal(query.JSON, &model); err != nil {
+		backend.Logger.Error(err.Error())
 		return nil, ErrorJSON
```

Comment:
probably it's better to return the error so the user (unprivileged) can get the information:

```go
		return nil, fmt.Errorf("%w: %v", ErrorJSON, err) 
```

Reply:
I was thinking about that too, it's better to return errors than log them. Done!